apps/escuelafrontend/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "next",
+  "root": true,
+  "rules": {
+    "react/prop-types": 0,
+    "react/no-unescaped-entities": 0
+  }
+}
```

Comment:
This fixed the ` jsx-a11y/anchor-is-valid` warning that was triggered on all `Links`

```
Warning: The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. 
```